shardingsphere-features/shardingsphere-readwrite-splitting/shardingsphere-readwrite-splitting-distsql/shardingsphere-readwrite-splitting-distsql-handler/src/test/java/org/apache/shardingsphere/readwritesplitting/distsql/handler/converter/ReadwriteSplittingRuleStatementConverterTest.java

```diff
@@ -0,0 +1,121 @@
+        assertSingleRuleSegmentConvertResult(singleReadwriteSplittingRuleSegment, singleRuleSegmentConvertResult);
+        Properties properties2 = new Properties();
+        properties2.setProperty("ping1", "pong1");
+        Properties properties3 = new Properties();
+        properties3.setProperty("ping2", "pong2");
+        Properties properties4 = new Properties();
+        properties4.setProperty("ping3", "pong3");
+        properties4.setProperty("ping4", "pong4");
+        List<ReadwriteSplittingRuleSegment> multipleReadwriteSplittingRuleSegments = Arrays.asList(
+                createReadwriteSplittingRuleSegment("name2", "write_ds_02", Arrays.asList("read_ds_02", "read_ds_03"), "lb02", properties2),
+                createReadwriteSplittingRuleSegment("name3", "write_ds_03", Arrays.asList("read_ds_04", "read_ds_05"), "lb03", properties3),
+                createReadwriteSplittingRuleSegment("name4", "write_ds_04", Arrays.asList("read_ds_06", "read_ds_07"), "lb04", properties4));
+        ReadwriteSplittingRuleConfiguration multipleRuleSegmentConvertResult = ReadwriteSplittingRuleStatementConverter.convert(multipleReadwriteSplittingRuleSegments);
+        assertMultipleReadwriteSplittingRuleSegments(multipleReadwriteSplittingRuleSegments, multipleRuleSegmentConvertResult);
+    }
+
+    private ReadwriteSplittingRuleSegment createReadwriteSplittingRuleSegment(final String name,
+                                                                              final String writeDataSource,
+                                                                              final List<String> readDataSourceList,
+                                                                              final String loadBalancerTypeName,
+                                                                              final Properties properties) {
+        return new ReadwriteSplittingRuleSegment(name, writeDataSource, readDataSourceList, loadBalancerTypeName, properties);
+    }
+
+    private void assertEmptyRuleSegmentConvertResult(final ReadwriteSplittingRuleConfiguration emptyRuleSegmentConvertResult) {
+        assertTrue(emptyRuleSegmentConvertResult.getDataSources().isEmpty());
+        assertTrue(emptyRuleSegmentConvertResult.getLoadBalancers().isEmpty());
+    }
+
+    private void assertSingleRuleSegmentConvertResult(final ReadwriteSplittingRuleSegment readwriteSplittingRuleSegment,
+                                                      final ReadwriteSplittingRuleConfiguration singleRuleSegmentConvertResult) {
+        Collection<ReadwriteSplittingDataSourceRuleConfiguration> singleRuleSegmentConvertResultDataSources = singleRuleSegmentConvertResult.getDataSources();
+        Map<String, ShardingSphereAlgorithmConfiguration> singleRuleSegmentConvertResultLoadBalancers = singleRuleSegmentConvertResult.getLoadBalancers();
+        assertEquals(1, singleRuleSegmentConvertResultDataSources.size());
```

Comment:
Please use `assertThat` to instead of `assertEquals`

Reply:
ok, already edited